File: _GameNN/Scripts/GlobalVar.cs
using UnityEngine;
using System.Collections;

public class GlobalVar {

	public static string Top = "top";
	public static string Left = "left";
	public static string Right = "right";


	public enum PlayerPosition {
		Down,
		RightDown,
		Right,
		RightUp,
		Top,
		LeftUp,
		Left,
		LeftDown,
	}

	/// <summary>
	/// 游戏界面肤色
	/// </summary>
	public static string SKIN_COLOR = "green";
    public static string SKINCARD_COLOR = "blue";
}

File: Lua/Common/GlobalVar.lua

local this = LuaObject:New()
GlobalVar = this 

this.Top = "top";
this.Left = "left";
this.Right = "right";


this.PlayerPosition =  {
	['Down'] = "Down",
	['RightDown'] = 'RightDown',
	['Right'] = 'Right',
	['RightUp'] = 'RightUp',
	['Top'] = 'Top' ,
	['LeftUp'] = 'LeftUp',
	['Left'] = 'Left',
	['LeftDown'] ='LeftDown' ,
}

-- 游戏界面肤色
this.SKIN_COLOR = "green";
this.SKINCARD_COLOR = "blue";